src/Components/MediaPresenter.ts

```diff
@@ -68,55 +56,41 @@ export class MediaPresenter {
   }
 
   @mobx.action
-  setDuration(index: number, duration: number): void {
-    this.durations[index] = duration;
+  setDuration(fileIndex: number, duration: number): void {
+    this.durations[fileIndex % this.filesLength] = duration;
   }
 
-  getDuration(index: number, styleIndex: number): number {
-    if (this.customOrder) {
-      return this.durations[index % this.filesLength];
-    }
-    return this.durations[this.seeds[styleIndex][index]];
+  getDuration(fileIndex: number): number {
+    return this.durations[fileIndex % this.filesLength];
   }
   // only called in import modal, no seeding needed
-  getFileName(index: number): string {
-    return this.files[index].name;
-  }
-  getPreviewFile(fileIndex: number): File {
-    return this.files[fileIndex];
-  }
-  getFile(mediaCounter: number, styleIndex: number): File {
-    if (this.customOrder) {
-      return this.files[mediaCounter % this.filesLength];
-    }
-    return this.files[this.seeds[styleIndex][mediaCounter]];
+  getFileName(fileIndex: number): string {
+    return this.files[fileIndex % this.filesLength].name;
   }
-  getCurrFile(): File {
-    return this.files[this.currFileIndex];
+  getFile(fileIndex: number): File {
+    return this.files[fileIndex % this.filesLength];
   }
-
+  //call it by: mediaPresenter.filesLength
   @mobx.computed
   get filesLength(): number {
     return this.files.length;
   }
 
   @mobx.action
-  addFilePlayed(fileIndex: number, playedDur: number): void {
-    this.played[fileIndex] =
-      (this.played[fileIndex] + playedDur) % this.durations[fileIndex];
+  increFilePlayed(fileIndex: number):void {
```

Comment:
nit: i'd probably rename this to be `setFilePlayed` or full word `incrementFilePlayed`